src/main/java/com/carmen/app/entities/Car.java

```diff
@@ -25,38 +31,37 @@
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private UUID id;
 	
-	@Column(name = "BRAND_CAR")
+	@Column(name = "brand_car")
 	private String brand;
 	
-	@Temporal(TemporalType.TIMESTAMP)
-	@Column(name = "Registration date")
-	private Date registration;
+	//@Temporal(TemporalType.TIMESTAMP)
+	@Column(name ="registration_date")
+	private LocalDateTime registration;
 	
-	@Column(name = "Country")
+	@Column(name = "country")
 	private String country;
 	
-	@Temporal(TemporalType.TIMESTAMP)
-	@Column(name = "Created")
-	private Date created_at;
+	//@Temporal(TemporalType.TIMESTAMP)
+	@Column(name = "created")
+	@CreationTimestamp
+	private LocalDateTime created_at;
```

Comment:
For fields compose of multiple words it is better to use [camel case](https://en.wikipedia.org/wiki/Camel_case) instead of [snake case](https://en.wikipedia.org/wiki/Snake_case).
Snake in java is mostly used for constant names.